salt/_pillar/metalk8s_nodes.py

```diff
@@ -92,9 +96,42 @@ def ext_pillar(minion_id, pillar, kubeconfig):
             for node in node_list.items
         )
 
+        volume_information = []
+
+        try:
```

Comment:
If this were put in a separate function, there'd be no need for a massive `try`/`except` block (and the subsequent indent).